src/utils.ts

```diff
@@ -135,3 +135,27 @@ export const toBase64 = (payload: string): string => {
 
     throw new Error('no base64 encoding method available');
 }
+
+export const retryUntilPagination = <T>(fn: (...args: any) => Promise<SearchResponse<T>>) => {
```

Comment:
I think a function name of `fetchAll` instead of `retryUntilPagination` makes more sense

Reply:
done. Renamed to `fetchAll`